libcudacxx/test/libcudacxx/std/cccl/builtin.bit.clz.fail.cpp

```diff
@@ -0,0 +1,21 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+// SPDX-FileCopyrightText: Copyright (c) 2025 NVIDIA CORPORATION & AFFILIATES.
+//
+//===----------------------------------------------------------------------===//
+
+#include <cuda/std/__cccl/builtin.h>
+
+int main(int, char**)
+{
+#if !defined(_CCCL_BUILTIN_CLZLL)
+  auto x = _CCCL_BUILTIN_CLZLL(0b10101010);
```

Comment:
Important: This file is called `...clz.fail` (without `ll`) but you test the intrinsic `_CCCL_BUILTIN_CLZLL`. And vice versa in the other file. Please test the `_CCCL_BUILTIN_CLZ` here and `_CCCL_BUILTIN_CLZLL` in `builtin.bit.clzll.fail.cpp`.